azureDatabaseUtils/src/test/java/bio/terra/workspace/azureDatabaseUtils/BaseUnitTest.java

```diff
@@ -0,0 +1,13 @@
+package bio.terra.workspace.azureDatabaseUtils;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ActiveProfiles("test")
+@SpringBootTest
+@ContextConfiguration(classes = AzureDatabaseUtilsApplication.class)
+@ExtendWith(SpringExtension.class)
+public abstract class BaseUnitTest {}
```

Comment:
Seems it is used more like integration test... Might be confusing for those who would write unit tests. Full application context is loaded.. Please correct me if I am wrong.

Reply:
seems to be pretty standard https://github.com/DataBiosphere/terra-workspace-manager/blob/main/service/src/test/java/bio/terra/workspace/common/BaseTest.java